dataplane/external/ext_dataplane.go

```diff
@@ -33,6 +34,9 @@ import (
 // StartExtDataplaneDriver starts the given driver as a child process and returns a
 // connection to it along with the command itself so that it may be monitored.
 func StartExtDataplaneDriver(driverFilename string) (*extDataplaneConn, *exec.Cmd) {
+	var wg sync.WaitGroup
+	var errStdout, errStderr error
```

Comment:
Please move these to where they're needed.  Minimising the scope of variables is best practice (and we don't need to worry about archaic C compilers since we're not writing C)

Reply:
Fixed